### PR TITLE
Update README.md with development information

### PR DESCRIPTION
**Pull request by moosecanswim**
updates the readme with development versioning informaiton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,35 @@ Integration tests are run automatically upon pull requests; however, you can emu
...
+## Development
+
+Current Release Branch: `release-2.3`
+Future Release Branch: `release-2.4`
+
+>*If changes are made to the current release branch make sure to ensure corresponding changes are made to the future release branch*
+
+### Versioning
+
+GreyMatter Helm Charts follow semver principals and allow us to manage active development for current and future releases.  Pre-release charts will have a suffix of `-x` which indicates to helm that this is a prerelease chart.  These are not used for installation or dependency fulfillment unless specifically called out.
```
Comment (sofiashalotenko) [inline README.md:82]: Grey (space) Matter, I think.

File: README.md [hunk 2]
```diff
@@ -70,6 +70,35 @@ Integration tests are run automatically upon pull requests; however, you can emu
...
+
+*Increment Major:*
+If a helm release can not be upgraded using a `helm upgrade <release_name>`
+Adding or major charts
+
+*Increment Minor:*
+Addition, removal, substitution of values files and/or  templates have changes that make them incompatible with older values files
+
+*Increment Patch:*
+Template logic changes which do not result in modifications to values files (are backwards comparable)
```
Comment (sofiashalotenko) [inline README.md:96]: did you mean backwards compatible?

File: README.md [hunk 3]
```diff
@@ -70,6 +70,35 @@ Integration tests are run automatically upon pull requests; however, you can emu
...
+Any change to the Helm Chart templates or values files will require the chart version to be incremented in order to pass ci/cd linting.
+
+**General guidance for release/ tag versioning:**
+
+*Increment Major:*
+If a helm release can not be upgraded using a `helm upgrade <release_name>`
+Adding or major charts
+
+*Increment Minor:*
+Addition, removal, substitution of values files and/or  templates have changes that make them incompatible with older values files
```
Comment (sofiashalotenko) [inline README.md:93]: small nitpick...I think you have an extra space at "and/or  templates"

File: README.md (hunk 2 above)
Comment (moosecanswim) [reply README.md]: yup

**Review by sofiashalotenko — APPROVED** [review]
lgtm

**Pull request merged**